test/concurrent.cc

```diff
@@ -470,29 +533,30 @@ TEST(ConcurrentTest, InterruptFailOrStop) {
 
   // NOTE: expecting "any" throwable here depending on whether the
   // eventual that stopped or failed was completed first.
-  EXPECT_ANY_THROW(future.get());
+  // Expecting 'std::exception_ptr' for 'ConcurrentOrdered'.
+  this->template ExpectAnyOr<std::exception_ptr>(std::move(future));
```

Comment:
Let's just be explicit here, we can use `constexpr` here (and above where you did this).